src/constants.js

```diff
@@ -38,9 +38,11 @@ const SEARCH_RESULTS_PER_PAGE = 23;
 
 const AUTOMATIC_CREATION = 'Automatic';
 const CURATED_CREATION = 'Curated';
+const SYSTEM_GENERATED = 'System Generated';
```

Comment:
Why this is an idea, the creationMethod column is for where the template came from, not how it is used. I think a new column would better serve what your trying to do here.

Maybe something like canUserInstantiate would be better. For all current rows its true and for this one row its false.

Reply:
its temporary for now to prevent using the new goal template.